runtime/src/rpc.rs

```diff
@@ -1611,6 +1613,23 @@ impl VaultRegistryPallet for InterBtcParachain {
 
         Ok(result.amount)
     }
+
+    async fn get_collateralization_from_vault(&self, vault_id: VaultId, only_issued: bool) -> Result<u128, Error> {
```

Comment:
Do we still use this RPC in the lib? Wondering if we should just do the calculation here since we anyway read the storage values.

Reply:
The logic is duplicated in the lib because of issues with RPC calls in polkadot-js. We should revert to using the RPC call once fixed, so best to keep it in the parachain.